content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -79,14 +79,15 @@ following steps:
 
    See [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join/) for additional info.
 
-1. Installs the internal DNS server (kube-dns) and the kube-proxy addon components via the API server. If kubeadm is invoked with --feature-gates=CoreDNS=true, then [CoreDNS](https://coredns.io/) will be installed as the default internal DNS server instead of kube-dns.  
+1. Installs a DNS server (CoreDNS) and the kube-proxy addon components via the API server.
+   In Kubernetes version 1.11 and later CoreDNS has become the default DNS server.
+   To install kube-dns instead of CoreDNS, `kubeadm` has to be invoked with `--feature-gates=CoreDNS=false`.
```

Comment:
s/`kubeadm`/kubeadm (we're not consistent, but let's move toward this standard)
s/has to/must